routes/users.js

```diff
@@ -21,6 +21,13 @@ router.get("/:userId/intro", authenticate, authorizeRoles([SUPERUSER]), users.ge
 router.put("/self/intro", authenticate, userValidator.validateJoinData, users.addUserIntro);
 router.get("/:id/skills", users.getUserSkills);
 router.get("/:id/badges", getUserBadges);
+router.patch(
+  "/:userId/roles",
+  authenticate,
+  userValidator.validateUpdateRoles,
+  authorizeRoles([SUPERUSER]),
```

Comment:
super user validator can be done before validateUpdateRoles so that we need not validate the body if the user is not a super user